easybuild/easyconfigs/s/Singularity/Singularity-2.5.1-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'Singularity'
+version = '2.5.1'
+
+homepage = 'http://singularity.lbl.gov'
+description = """Singularity is a portable application stack packaging and runtime utility."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+
+source_urls = ['https://github.com/singularityware/singularity/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['c1369d1ae0318876cf878eaa249d33f9699c6d77e0cdb2c55522f6714beb306e']
+
+builddependencies = [('Autotools', '20150215')]
+dependencies = [('libarchive', '3.3.1')]
```

Comment:
@bilke `libarchive` is only a *build* dependency I think?

Reply:
Right!